src/components/Slide/Slide.tsx

```diff
@@ -0,0 +1,32 @@
+import CardSlide from '@Component/CardSlide';
+import styles from './slide.module.css';
+
+type SlideProps = {
+  data: CardProps[]; // data that the card component needs
+  title: string; // section title
+  height?: string; // optional slide height
+}
+
+const Slide = ({data, title, height = '200px'}: SlideProps) => {
+  return (
+    <>
+      <h2 className={styles.slide__title}>{title}</h2>
+      <div className={styles.slide} >
+        <CardSlide data={data[0]} />
+        <CardSlide data={data[1]} />
+        <CardSlide data={data[2]} />
+        <CardSlide data={data[0]} />
+        <CardSlide data={data[1]} />
+        <CardSlide data={data[2]} />
```

Comment:
This is called `magic numbers` and `hardcoding`. Consider using a for loop to achieve the same at the same time of making the component more reusable.

Reply:
This is called `magic numbers` and `hardcoding`. Consider using a for loop to achieve the same at the same time of making the component more reusable.

Reply:
Yes, you're right. I had put those components like this because they are only for testing, when we have to create the page the cards will be of a single type and for that of course the map function would be used to make the component more reusable.

Reply:
Seems like a reasonable approach. In that case, I tested before and I think that your code is fine. We will merge once hardcoding is done.